exercise_7.py

```diff
@@ -47,10 +66,14 @@ def read_csv(file):
 	read_csv('AMZN.csv')
 	>>> 14
 	'''
-
+	df=read_csv(file)
```

Comment:
this works; however we are hoping to practice what we learned with `csv` module.